charts/zenko/values.yaml

```diff
@@ -28,30 +36,30 @@ cloudserver-front:
     keyId: deployment-specific-access-key
     secretKey: deployment-specific-secret-key
 
+backbeat:
+  mongodb:
+    replicas: *nodeCount
+
 prometheus:
   rbac:
     create: true
-
   alertmanager:
     enabled: false
-
   kubeStateMetrics:
     enabled: false
-
   nodeExporter:
     enabled: false
-
   pushgateway:
     enabled: false
 
 mongodb-replicaset:
   replicaSet: rs0
-  replicas: 3
+  replicas: *nodeCount
 
 zenko-queue:
 ## Extensive list of configurables can be found here:
 ## https://github.com/kubernetes/charts/blob/master/incubator/kafka/values.yaml
-  replicas: 3
+  replicas: *nodeCount
   rbac:
     enabled: true
   configurationOverrides:
```

Comment:
@ssalaues Do we need to set ```offsets.topic.replication.factor``` to nodeCount as well?

Reply:
Replication of 3 and insync of 2 should still provide enough HA for two failures. It's a user configurable option so if higher redundancy is needed.